chord_metadata_service/chord/ingest/experiments.py

```diff
@@ -164,12 +171,14 @@ def ingest_derived_experiment_results(json_data: list[dict]) -> list[em.Experime
     # First, validate all experiment results with the schema before creating anything in the database.
 
     for idx, exp_result in enumerate(json_data):
-        validation = schema_validation(exp_result, EXPERIMENT_RESULT_SCHEMA)
-        if not validation:
+        val_errors = schema_validation(exp_result, EXPERIMENT_RESULT_SCHEMA)
```

Comment:
`exp_result` would be the variable we need to transform if we want backwards-compatibility, right?

Reply:
Yes, I just added the missing arguments to generate warnings